packages/core/useThrottledRefHistory/index.ts

```diff
@@ -0,0 +1,24 @@
+import { MaybeRef, throttleFilter } from '@vueuse/shared'
+import { Ref } from 'vue-demi'
+import { UseRefHistoryOptions, UseRefHistoryReturn, useRefHistory } from '../useRefHistory'
+
+/**
+ * Same as [useRefHistory](https://vueuse.org/useRefHistory) but with throttle effect.
+ *
+ * @see https://vueuse.org/useThrottledRefHistory
+ * @param source
+ * @param options
+ * @param ms
+ */
+export function useThrottledRefHistory<Raw, Serialized = Raw>(
+  source: Ref<Raw>,
+  options: UseRefHistoryOptions<Raw, Serialized> = {},
+  ms: MaybeRef<number>,
```

Comment:
Same here, we could have the `ms` renamed to `throttle` and put inside of the options. Could have the type like: `options: Omit<UseRefHistoryOptions<Raw, Serialized>, 'filter'> & { throttle? MaybeRef<number> }`

Reply:
updated